pkgs/by-name/ps/ps3-disc-dumper/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -i bash -p bash nix nix-update git cacert
+set -eo pipefail
+
+nix-update ps3-disc-dumper
+$(nix-build . -A ps3-disc-dumper.fetch-deps --no-out-link)
```

Comment:
Can we do the `prev_version` thing here to skip `fetch-deps`?

I'd love to make `buildDotnetModule` automatically wrap `updateScript` to do this, but I haven't had time to work on it yet.

Reply:
> I'd love to make buildDotnetModule automatically wrap updateScript to do this

Or maybe it should be done in the nix-update repo? Seems like nix-update is searching for pnpm deps, cargo deps, etc., but not nuget deps.

Reply:
Maybe, if it could be done in a way that isn't too tightly bound to the implementation of `fetch-deps`.  I'd have to look at how those others work.